playbooks/installation/index.adoc

```diff
@@ -368,19 +385,19 @@ openshift_hosted_registry_storage_volume_size=100Gi
 For our HA OpenShift Cluster (`c1-ocp.myorg.com`), we will provision the following servers. The first in the list is referred to as the Ansible Control Host. We'll use that as a bastion VM from which we will access all of the other cluster hosts, run commands to configure the cluster, run the OpenShift installation playbooks, etc.
 
 * 1 Ansible control host (`control-host.myorg.com`)
-  ** RHEL 7.2 minimal installation
+  ** RHEL 7.3 minimal installation
```

Comment:
should we go ahead and start suggesting 7.4?

Reply:
Yes 7.4 is stable.  I have at least one customer using it when I left them

Reply:
I would prefer 7.4 but the docs show 7.3 or 7.4 will work.

Reply:
Updated.